tools/local-nodescanner/main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"os"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/stackrox/scanner/pkg/analyzer/nodes"
+)
+
+func main() {
+	flag.CommandLine = flag.NewFlagSet(os.Args[0], flag.ExitOnError)
+	fPath := flag.String("fspath", "/host", "Path to the root folder of a filesystem")
+	fRHCOSrequired := flag.Bool("rhcosRequired", true, "Fails scan if path does not contain a filesystem generated by RHCOS. Node Scanning only works on RHCOS.")
+	fUncertifiedRHEL := flag.Bool("uncertifiedRhel", false, "Set true for CentOS and false for RHEL.")
+	fVerbose := flag.Bool("verbose", false, "Print verbose output if set")
+	flag.Parse()
+
+	setupLog(*fVerbose)
+
+	fspath, err := filepath.Abs(*fPath)
+	if err != nil {
+		log.Fatalf("Encountered error while formatting path: %v", err)
+	}
+
+	log.Infof("Analyzing rootfs in %v", fspath)
+	components, err := nodes.Analyze(context.Background(), "nodename", fspath, nodes.AnalyzeOpts{UncertifiedRHEL: *fUncertifiedRHEL, IsRHCOSRequired: *fRHCOSrequired})
+	if err != nil {
+		log.Errorf("Encountered error while scanning: %v", err)
+	}
+
+	printResultsToTTY(components)
```

Comment:
Nit: You are simply writing to stdout, which might not be a TTY and can be some other file type. I would replace `TTY` with `Stdout` or just `printResults()`.